src/west/manifest.py

```diff
@@ -290,7 +290,7 @@ def __init__(self, name, remote, defaults, path=None, clone_depth=None,
         self.remote = remote
         self.url = remote.url + '/' + name
         self.path = path or name
-        self.abspath = os.path.normpath(os.path.join(util.west_topdir(), self.path))
+        self.abspath = os.path.realpath(os.path.join(util.west_topdir(), self.path))
```

Comment:
Looking at this, I agree that having normpath alone is not correct: `os.path.normpath('west/src')` on Windows gives me `west\src`, which is not an absolute path, while `os.path.realpath('west/src')` gives me `C:\Users\mbolivar\west\src` (i.e. it's absolutized and backslash-converted).

Care to throw in a call to `normcase` on the result while you're here? Maybe this is a separate issue, but we should probably forbid projects whose names are equal up to case, as that won't work on Windows...

Reply:
I won't work on Windows and also on macOS, depending on the installation options. That said, I do see this as a separate issue, maybe to be added to the error handling of special names in the manifest.

Reply:
Added a `normcase()`. Seems to be a no-op on platforms with case-sensitive names.

I'm not sure we should `normcase()` the `abspath` field itself. It might be nice to preserve the name as written there, if it's ever shown, and it could be normalized where it matters.

Reply:
Normalizing it in this particular situation seems reasonable, since you're using startswith as a proxy for containment.